src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/domain/move/Move.kt

```diff
@@ -133,6 +133,12 @@ data class Move(
   @Transient
   val person: Person? = null
 ) {
+  @Column(name = "move_month", nullable = true)
+  val moveMonth: Int? = moveDate?.month?.value
```

Comment:
Added columns in an effort to narrow down the data set of moves returned when querying moves for reporting purposes.